app/views/posts/_expanded.html.erb

```diff
@@ -315,7 +315,7 @@
 
         <div class="post--action-dialog js-flag-box">
           <div class="widget">
-            <div class="widget--header">Why does this post require moderator attention?</div>
+            <div class="widget--header">Why does this post require attention from curators or moderators?</div>
```

Comment:
I like the clarity here that both curators and moderators can see some flags.

Note that the pop up message still says "Thanks! A moderator will review your flag" regardless of whether it's a moderator only flag or a curator also flag.

Reply:
Reading this again, I hope it doesn't sound like a request for a context reactive message. This message that pops up is just an acknowledgement to confirm the flag was raised successfully, so it doesn't need to be specific about who will process it. I expect something along the lines of "Thanks! Someone will review your flag" or even "Thanks! We'll review your flag" would be fine to cover both cases.

Reply:
Don't worry; I didn't take it that way, and already changed it to "we".